### PR TITLE
add badges and change travis to do coveralls integration

### LINKED ISSUE #45: Add some badges
Travis CI, Coveralls, etc. ... :)

### PR DESCRIPTION
**Pull request by szuecs**
fix #45 to get some, hopefully working, badges

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
-  - 1.7
-  - tip
+- 1.7
+- tip
 
 before_install:
-  - go get github.com/Masterminds/glide
+- go get github.com/Masterminds/glide
+- go get github.com/mattn/goveralls
+- go get -u github.com/alecthomas/gometalinter
```
Comment (linki) [inline .travis.yml]: let's drop the `-u` as it shouldn't have any effect on travis (new container all the time)

File: .travis.yml [hunk 2]
```diff
@@ -1,18 +1,23 @@
...
 
 install:
-  - glide install --strip-vendor
+- glide install --strip-vendor
+- gometalinter --install
 
-script: make test
+script:
+- make test
+- goveralls -service=travis-ci
```
Comment (linki) [inline .travis.yml]: `goveralls` runs the tests itself somehow. I don't know how exactly. Let's find out so we don't run tests twice/get inconsistent results.

**Comment by szuecs** [comment]
:+1:

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**